c++/src/SchemaEvolution.cc

```diff
@@ -0,0 +1,259 @@
+#include "SchemaEvolution.hh"
+#include "orc/Exceptions.hh"
+
+#include <unordered_map>
```

Comment:
These two header files are already included in the SchemaEvolution.hh. It is a redundant code.

Reply:
donwe